BlueprintUI/Sources/ViewDescription/ViewDescription.swift

```diff
@@ -129,6 +133,14 @@ public struct ViewDescription {
 
 extension ViewDescription {
 
+    /// The available prioritization methods for rounding frames to pixel boundaries.
+    public enum RoundingPriority: Equatable {
```

Comment:
I would maybe call this `RoundingBehavior` since its behaviour / actions for this element; priority to me implies a relationship/ordering between other elements.

Reply:
I actually like priority, though I don't feel super strongly. But the cases are what you're prioritizing when rounding - either prioritizing keeping the size or the edges closer to the original.

Reply:
Yes sure – but IMO priority is a "other things outside me are also being considered"; where this purely applies to this thing; eg a behavior.

Reply:
what about
```swift
enum RoundingBehavior {
    case prioritizeSize
    case prioritizeEdges
}
```

Reply:
truly why not both

Reply:
> what about
> 
> ```swift
> enum RoundingBehavior {
>     case prioritizeSize
>     case prioritizeEdges
> }
> ```

I actually started with this, and the fact that both cases are "prioritize x" is what prompted me to instead describe it more simply in terms of x, the thing being prioritized. This value doesn't really describe the entire rounding behavior, it only describes which of these two things should be prioritized.

Reply:
updated to
```swift
    /// The available prioritization options for rounding frames to pixel boundaries.
    public enum FrameRoundingBehavior: Equatable {
        /// Prioritize preserving frame edge positions
        case prioritizeEdges
        /// Prioritize preserving frame sizes
        case prioritizeSize
    }
```